### PR TITLE
Examine text will now show how many items there is now in a stack

### LINKED ISSUE #7225: Examining stacks of items should tell you how many items are in each stack.
### Description
Title. The message in chat when you examine a stack of items should tell you how many items are in that stack.

### PR DESCRIPTION
**Pull request by MaxIsJoe**
title. closes #7225 

### Changelog:
CL: [New] - Stackable items' examine text now shows how many items they're holding.

### CODE REVIEW

**Comment by corp-0** [comment]
use the stackable component itself instead

**Comment by MaxIsJoe** [comment]
> use the stackable component itself instead

but.. that's what I'm doing?

**Comment by corp-0** [comment]
I meant, instead of adding a reference to Stackable and a check in Attributes component, implement IExaminable directly in Stackable component. Then you need no reference and no check. It is also cleaner because that string is only relevant for objects that are stackable.

**Comment by MaxIsJoe** [comment]
Done.

File: UnityProject/Assets/Scripts/Items/Stackable.cs [hunk 1]
```diff
@@ -341,4 +341,13 @@ public void ServerPerformInteraction(HandApply interaction)
 	{
 		ServerCombine(interaction.TargetObject.GetComponent<Stackable>());
 	}
+
+	public string Examine(Vector3 worldPos)
+	{
+		string displayName = gameObject.ExpensiveName();
+
+		string str = "This " + displayName + " contains (" + Amount + ") stacks.";
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Items/Stackable.cs]:
```suggestion
		return $"This {gameObject.ExpensiveName()} contains {Amount} stacks.";
```
I think it is fine without the variables.

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Items/Stackable.cs]: added

**Comment by MaxIsJoe** [comment]
fixed

File: UnityProject/Assets/Scripts/Items/Stackable.cs [hunk 2]
```diff
@@ -341,4 +341,9 @@ public void ServerPerformInteraction(HandApply interaction)
 	{
 		ServerCombine(interaction.TargetObject.GetComponent<Stackable>());
 	}
+
+	public string Examine(Vector3 worldPos)
+	{
+		return $"This {gameObject.ExpensiveName()} contains {Amount} stacks.";
```
Comment (NoooneyDude) [inline UnityProject/Assets/Scripts/Items/Stackable.cs:347]: "This `metal sheet` contains 20 `stacks`."? "It contains 20 items." would be better.

**Pull request merged**